packages/pyright-internal/src/common/configOptions.ts

```diff
@@ -423,6 +423,9 @@ export interface DiagnosticRuleSet {
     reportUnusedParameter: DiagnosticLevel;
     reportImplicitAbstractClass: DiagnosticLevel;
     reportUnannotatedClassAttribute: DiagnosticLevel;
+
+    // What libraries' functions can be called even if they aren't typed?
```

Comment:
is this comment just documenting  this new setting? if so i would suggest removing it as it will be sufficiently documented in other, more user-facing places anyway which makes these comments redundant (i left the upstream ones there to avoid unnecessary conflicts)

```suggestion
```